src/test/java/org/myrobotlab/integration/InMoovScriptTest.java

```diff
@@ -115,6 +119,7 @@ public void testInMoovMinimalHead() throws IOException {
   // Test the inmoov minimal arm script.
   @Test
   public void testInMoovMinimalTorso() throws IOException {
+    if (printMethods)System.out.println(String.format("Running %s.%s", getSimpleName(), getName()));
```

Comment:
this sort of debug message should be refactored into the base class... or removed altogether.